integration/kubernetes/k8s-copy-file.bats

```diff
@@ -33,6 +33,24 @@ setup() {
 	kubectl exec $pod_name -- sh -c "cat /tmp/$file_name | grep $content"
 }
 
+@test "Copy from pod to host" {
+	# Create pod
+	kubectl create -f "${pod_config_dir}/pod-env.yaml"
+
+	# Check pod creation
+	kubectl wait --for=condition=Ready pod "$pod_name"
+
+	# Create a file in the pod
+	kubectl exec $pod_name -- sh -c "cd /tmp && echo $content > $file_name"
+
+	# Copy file from pod to host
+	kubectl cp $pod_name:tmp/$file_name $file_name
```

Comment:
tiny nit: should that be `/`tmp/$file_name ?

Reply:
please also quote variables

Reply:
@chavafg changes applied